extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -86,7 +86,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.type_ = null;
 
     /** @private {!boolean} */
-    this.isSandbox_ = element.hasAttribute('sandbox');
+    this.isSandbox_ = false;
```

Comment:
What does sandbox actually do here?
Should we assume true until we know otherwise?